halo2-base/src/lib.rs

```diff
@@ -324,6 +403,7 @@ impl<F: ScalarField> Context<F> {
         self.last().unwrap()
     }
 
+    /// Assigns the 0 value from `zero_cell` and returns the last assigned cell of the `advice` column.
```

Comment:
Either assigns `0` to a new cell or returns a previously assigned zero cell. 

Does not need to be last assigned cell